src/components/BlockingViews/FullPageNotFoundView.js

```diff
@@ -24,11 +28,21 @@ const defaultProps = {
 const FullPageNotFoundView = (props) => {
     if (props.shouldShow) {
         return (
-            <BlockingView
-                icon={Expensicons.QuestionMark}
-                title={props.translate('notFound.notHere')}
-                subtitle={props.translate('notFound.pageNotFound')}
-            />
+            <View style={[styles.flex1, styles.dFlex, styles.flexColumn]}>
+                <HeaderWithCloseButton
+                    shouldShowBackButton
+                    onBackButtonPress={() => Navigation.dismissModal()}
+                    onCloseButtonPress={() => Navigation.dismissModal()}
```

Comment:
Here I think we should use `Navigation.navigate(ROUTES.HOME)` instead because I believe there could be multiple modals, so if we dismiss one it will show the one beneath.

Reply:
Can you provide an example of when we have multiple modals open? is the LHN a "modal"? (sorry if this is obvious, but I'm not that familiarized with this)

Reply:
Hmm, after playing with it a bit more I think I'm probably wrong here. I think only one modal can be shown at a time from the root stack navigator in AuthScreens. Also, `Navigation.dismissModal()` calls [navigateBackToRootDrawer](https://github.com/Expensify/App/blob/c5fb8777a18ddc5b3e367d21fc012a474d3760c3/src/libs/Navigation/CustomActions.js#L23) so it should be fine.

We can stick with this.